include/winsparkle.h

```diff
@@ -242,6 +242,38 @@ WIN_SPARKLE_API void __cdecl win_sparkle_set_app_build_version(const wchar_t *bu
  */
 WIN_SPARKLE_API void __cdecl win_sparkle_set_registry_path(const char *path);
 
+/// Type used to override WinSparkle's config read function
+typedef int(__cdecl *win_sparkle_config_read_t)(void *custom_data, const char *name, wchar_t *buf, size_t bytes);
+/// Type used to override WinSparkle's config write function
+typedef int(__cdecl *win_sparkle_config_write_t)(void *custom_data, const char *name, const wchar_t *value);
```

Comment:
inconsistent naming
`buf` and `value` are the same entity
is there a reason why you would name them differently?

Reply:
@Youw That's clearly copied from the existing code, and I wouldn't consider it a problem: the read function *is* reading into a buffer and if anything, it emphasizes the need to specify the buffer's length.

Reply:
Hmmm... Because it doesn't make sense that users needs to "provide a value" when they need to read the value - which means they don't have a value yet, they can only provide a buffer to receive value.

Reply:
maybe you're right
just my own observation - fresh look without a context